internal/indexer/indexer.go

```diff
@@ -38,6 +40,8 @@ const (
 	StagesByWarehouseField      = "warehouse"
 
 	ServiceAccountsByOIDCClaimsField = "claims"
+
+	WarehouseRepoURLIndexKey = "subscriptions.repoURL"
```

Comment:
This feels like it would be a bit more consistent with those above:

```suggestion
	WarehousesBySubscribedURLsField = "subscribedURLs"
```